gradle.properties

```
@@ -28,3 +28,10 @@ org.gradle.configuration-cache=false
 #org.gradle.configuration-cache-problems=warn
 # bump the Gradle daemon heap size (you can set bigger heap sizes as well)
 org.gradle.jvmargs=-Xms2g -Xmx4g -XX:MaxMetaspaceSize=768m
+
+# Matrix testing configuration for integration tests
+# Comma-separated list of versions to test against
+# The last version in the list is the default and will be used for the main intTest task
+authmgr.test.iceberg.versions=1.9.0,1.9.1
```

```suggestion
authmgr.test.iceberg.versions=1.9.0,1.9.2
```
Do we need all 1.9 patch versions?

---

I don't think so, also 1.10 is almost there so I would just retain 1.10 and 1.9.2 when that happens.